local-proxy/README.md

```diff
@@ -0,0 +1,47 @@
+Flipt Local Proxy
+------------
+
+The intent of this lab is to explore the ways that clients can achieve fast evaluations from Flipt for their feature flags. The use case here is that users would like data locality for their feature flag state, so they do not have to worry about network latency to evaluate a feature, especially for their most critical application paths.
+
+### Object Store Replication
+
+<img src="./object-store-replication/diagrams/diagram.svg" alt="Object Store Replication" width="500px" />
+
+This project is housed under the `object-store-replication` directory. The purpose is for a user to run their application which depends on feature flags with a sidecar Flipt process that pulls data from an Object Store (S3 bucket). The evaluation data here can then be accessed over `localhost` by talking to the Flipt sidecar.
+
+Lets get started!
+
+Prerequisites:
+- [Docker](https://www.docker.com/)
+- [Kubectl](https://kubernetes.io/docs/reference/kubectl/)
+- [Minikube](https://minikube.sigs.k8s.io/docs/)
+
+This project uses Kubernetes to deploy the following:
+- `minio`: Object store that has an S3 compatible API
+- `flipt-master`: Serves as the main Flipt application, this is where users will be accessing the UI to make relevant changes
+- `sample-app`: Serves as the pod with Flipt running as a sidecar, `flipt-sidecar`. There is also a container called `evaluation-client` that will make evaluation calls to the sidecar
+- `flipt-exporter`: CronJob that runs on a 1-minute interval that exports data out of the Flipt master, and puts those changes into the object store
+
+To access the `flipt-master` API, you can use Kubernetes to port-forward the service:
```

Comment:
im wondering if we should move these instructions under the bit that talks about running `scripts/start`, as going through this, once I run `scripts/start` im not sure what im actually supposed to do next.

Maybe restructuring these as steps 1., 2., 3. etc and moving them undre the `scripts/start` would help?

Reply:
@markphelps Agreed. Can do that.